07LessonSeven/homework/css/style.css

```diff
@@ -74,6 +74,8 @@ p {
   background-color: #55a747
 }
 
```

Comment:
One critique, you used and ID of `#button` for two elements, remember that you can only use one ID for a single element, IDs uniquely identify ONLY one element. See where you have `<div id="button">` in two locations in your HTML markup. You should have used a class `.button` instead